pyk/src/pyk/kast/outer.py

```diff
@@ -1519,8 +1519,8 @@ def _wrap_elements(_k: KInner) -> KInner:
             return _k
 
         # To ensure we don't get duplicate wrappers.
-        _kast = self.remove_cell_map_items(kast)
-        return bottom_up(_wrap_elements, _kast)
+        _kast = bottom_up(_wrap_elements, kast)
+        return self.remove_cell_map_items(_kast)
 
```

Comment:
This change in particular, is _my guess_ about what is causing the CI failures. If you need this change, please add a test (probably to `test_kast_to_kore.py`) that demonstrates why it's needed. Currently the failing test is in that category: `TestKonvertCellMap.test_kast_to_kore[account-cell-map]`.